Tests/PHPCI/Helper/AnsiConverterTest.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace PHPCI\Helper\Tests;
+
+use PHPCI\Helper\AnsiConverter;
+use Prophecy\PhpUnit\ProphecyTestCase;
+
+class AnsiConverterTest extends ProphecyTestCase
```

Comment:
Since we are not using any phpunit functionality, the last version of phpunit is shipped with Prophecy, i prefer to use the ```\PHPUnit_Framework_TestCase``` class as base test instead of ```ProphecyTestCase```.